docs/articles_en/openvino_workflow/running_inference_with_openvino/string_tensors.rst

```diff
@@ -0,0 +1,217 @@
+
+`ov::Tensor` allocates and initializes required number of `std::string` objects under the hood.
+
+
+Accessing Elements
+##################
+
+The code below prints all elements in 1D string tensor constructed above.
+In C++ code the same `.data` template method is used for other data types,
+and to access string data it should be called with `std::string` type.
+In Python, dedicated fields `std_data` or `byte_data` are used instead of `data` field for numerical data.
+
+.. tab-set::
+
+   .. tab-item:: Python
+      :sync: py
+
+      .. code-block:: python
+         :force:
+
+         data = tensor.str_data  # use tensor.byte_data instead to access encoded strings as `bytes`
+         for i in range(tensor.get_size()):
+            print(data[i])
+
+   .. tab-item:: C++
+      :sync: cpp
+
+      .. code-block:: cpp
+
+         #include <iostream>
+
+         std::string* data = tensor.data<std::string>();
+         for(size_t i = 0; i < tensor.get_size(); ++i)
+            std::cout << data[i] << '\n';
+
+In the case of Python, an object retrieved with `tensor.str_data` (or `tensor.bytes_data`) is numpy array
+with elements `numpy.str_` (or `numpy.bytes_` correspondingly). It is a copy of underlying data from
+the `tensor` object and cannot be used for tensor content modification.
+To set new values, the entire tensor content should be set as a list or as a `numpy` array as demonstrated
+below.
+
+In contrast to Python, when using `tensor.data<std::string>()` in C++, a pointer to underlying data
+storage is returned and it can be used for tensor element modification:
+
+.. tab-set::
+
+   .. tab-item:: Python
+      :sync: py
+
+      .. code-block:: python
+
+         # Unicode strings:
+         tensor.str_data = ['one', 'two', 'three']
+         # Do NOT use tensor.str_data[i] to set a new value, it won't update the tensor content
+
+         # Encoded strings:
+         tensor.bytes_data = [b'one', b'two', b'three']
+         # Do NOT use tensor.bytes_data[i] to set a new value, it won't update the tensor content
+
+      :sync: cpp
+
+      .. code-block:: cpp
+
+         std::string new_content[] = {"one", "two", "three"};
+         std::string* data = tensor.data<std::string>();
+         for(size_t i = 0; i < tensor.get_size(); ++i)
+            data[i] = new_content[i];
+
+When reading or setting string tensor elements in Python, it is recommended to use `str` objects (or `numpy.str_` if used in numpy array)
+when it is known that an underlying byte sequence forms a valid UTF-8 encoded string.
+Otherwise, if arbitrary byte sequences are allowed,
+not mandatory UTF-8 encoded, use `bytes` strings (or `numpy.bytes_` correspondingly) instead.
+
+Accessing tensor content through `str_data` implicitly applies UTF-8 decoding.
+If some parts of the byte stream cannot be represented as a valid Unicode symbol,
+the replacement symbol � is used to signal errors in such invalid Unicode streams.
```

Comment:
```suggestion
the replacement symbol of "�" is used to signal errors in such invalid Unicode streams.
```

Reply:
That � symbol _is_ a replacement symbol. Is `of` still required?

Reply:
It is similar to the case of fields, above. We can change it to "The � replacement symbol is used to..."